framework/src/problems/EigenProblem.C

```diff
@@ -392,6 +398,11 @@ EigenProblem::initEigenvector(const Real initial_value)
   // We, in general, do not need to worry about that.
   for (auto & vn : var_names)
   {
+    // There is nothing we need to scale. We do not need to return here, we need
```

Comment:
I would change "do not need to" to "cannot"

Reply:
Changed